Formula/openmodelica.rb

```diff
@@ -0,0 +1,94 @@
+class Openmodelica < Formula
+  desc "Open-source modeling and simulation tool"
+  homepage "https://openmodelica.org/"
+  # GitHub's archives lack submodules, must pull:
+  url "https://github.com/OpenModelica/OpenModelica.git",
+    revision: "11fcab4f2d6895f2db073572b2bff1a43177313f", tag: "v1.16.5"
```

Comment:
```suggestion
  url "https://github.com/OpenModelica/OpenModelica.git",
      tag:      "v1.16.5",
      revision: "11fcab4f2d6895f2db073572b2bff1a43177313f"
```
Let's follow the style of other formulae here.

Reply:
You might want to update the [Formula Cookbook](https://docs.brew.sh/Formula-Cookbook).  It had recommendations like `:revision =>` instead of `revision:`.